server/src/database/index.ts

```diff
@@ -0,0 +1,34 @@
+// SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import { Sequelize } from 'sequelize';
+import dbCreds from '../configs/database';
+
+export const sequelize = new Sequelize(dbCreds.database, dbCreds.username, dbCreds.password, {
+  host: dbCreds.host,
+  dialect: 'postgres',
+  dialectOptions: {
+    ssl: {
```

Comment:
Could you elaborate more on using TLS here, and the network structure? What creates the need for `rejectUnauthorized: false`?

Reply:
rejectUnauthorized can be removed, had it temporary false for dev purposes. defaults to true